user-access-handlers/src/test/java/no/unit/nva/handlers/IdentityServiceInitHandlerTest.java

```diff
@@ -141,10 +148,27 @@ void shouldCreateSiktCustomer() throws NotFoundException, IOException {
         assertThat(customer.getCristinId(), is(equalTo(SIKT_CRISTIN_ID)));
     }
 
+    @Test
+    void shouldCreateSiktBackendClientDBRow() throws NotFoundException, IOException {
+        var handler = new IdentityServiceInitHandler(identityService, customerService, ROLE_SOURCE);
+        handler.handleRequest(createRequest(), output, context);
+        var client = identityService.getClient(ClientDto.newBuilder().withClientId(BACKEND_CLIENT_ID).build());
+        assertThat(client, is(not(nullValue())));
+        assertThat(client.getActingUser(), is(equalTo(SIKT_ACTING_USER)));
+    }
+
+    @Test
+    void shouldNotCreateDuplicateSiktBackendClientDBRow() throws IOException {
+        var handler = new IdentityServiceInitHandler(identityService, customerService, ROLE_SOURCE);
+        handler.handleRequest(createRequest(), output, context);
+        handler.handleRequest(createRequest(), output, context);
+        verify(identityService, atMostOnce()).addExternalClient(any());
```

Comment:
maybe also assert that nothing was thrown?

Reply:
not sure how. addExternalClient call is inside attempt and if both get and create fails, the handle and test will fail as far as I can see